juice-examples/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 
 [dependencies]
 greenglas = { version = "0.2" }
-juice = { version = "0.2", default-features = false }
+juice = { version = "0.2.3", default-features = false }
```

Comment:
this should not be necessary, can you try a `cargo update` in the examples dir?

Reply:
Oh cool, it did remove it. No idea why I hadn't found that before